src/Sound.js

```diff
@@ -87,20 +87,22 @@ export default function Sound(props) {
     };
     fetchSound();
   }, [id, freeSound]);
-
+  /* eslint-disable-next-line */
   useEffect(() => {
-    const unsubscribe = tinykeys(window, {
-      Space: (event) => {
-        event.preventDefault();
-        setIsPlaying((i) => !i);
-      },
-      d: () => {
-        if (isLoggedIn) downloadSound(sound);
-      },
-    });
-    return () => {
-      unsubscribe();
-    };
+    if (document.activeElement.id === "body") {
```

Comment:
no need to have an id here. There should only be one body. What you could do is check for reference equality: `document.activeElement === document.body`

Reply:
Ah, I was not aware of that. Thanks for the tip!